serde-oracle-jdbc-json/src/test/java/io/micronaut/serde/bson/SampleData.java

```diff
@@ -0,0 +1,114 @@
+package io.micronaut.serde.bson;
+
+import io.micronaut.core.annotation.Introspected;
+import io.micronaut.serde.annotation.Serdeable;
+import io.micronaut.serde.oracle.jdbc.json.serde.OracleJsonDateSerde;
+import io.micronaut.serde.oracle.jdbc.json.serde.OracleJsonDurationSerde;
+import io.micronaut.serde.oracle.jdbc.json.serde.OracleJsonInstantSerde;
+import io.micronaut.serde.oracle.jdbc.json.serde.OracleJsonLocaleDateTimeSerde;
+import io.micronaut.serde.oracle.jdbc.json.serde.OracleJsonOffsetDateTimeSerde;
+import io.micronaut.serde.oracle.jdbc.json.serde.OracleJsonPeriodSerde;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.OffsetDateTime;
+import java.time.Period;
+import java.util.Date;
+import java.util.UUID;
+
+@Introspected
+public class SampleData {
+
+    @Serdeable.Deserializable(using = OracleJsonLocaleDateTimeSerde.class)
+    @Serdeable.Serializable(using = OracleJsonLocaleDateTimeSerde.class)
+    private LocalDateTime localDateTime;
+
+    @Serdeable.Deserializable(using = OracleJsonOffsetDateTimeSerde.class)
+    @Serdeable.Serializable(using = OracleJsonOffsetDateTimeSerde.class)
+    private OffsetDateTime offsetDateTime;
+
+    @Serdeable.Deserializable(using = OracleJsonDateSerde.class)
+    @Serdeable.Serializable(using = OracleJsonDateSerde.class)
+    private Date date;
+
+    @Serdeable.Deserializable(using = OracleJsonInstantSerde.class)
```

Comment:
This is how we need to configure fields if we want to read from Oracle JSON and be able to write back to. It means these beans might require DTO for data transfer over web because it uses serialization for DATA json.